src/modal.css

```diff
@@ -1,7 +1,7 @@
 .sc-modal {
     display: none;
     position: fixed; /* Stay in place */
-    z-index: 1; /* Sit on top */
+    z-index: 4000; /* Sit on top */
```

Comment:
Haha, modals are always having zindex issues 😆

Reply:
This keeps the modal 'close' button on the screen at all times so the user can always close out even if they scrolled. The design doc's implementation is much prettier so hopefully we can implement that as well